tools/eslint-config/index.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
     {
       files: ['*.ts', '*.tsx'],
       extends: ['./rules/base', './rules/typescript', './rules/import', './rules/prettier'].map(require.resolve),
+      plugins: ['import'],
```

Comment:
This line should be moved to `rules/import.js` file (I tested it)

Reply:
I see...thank you!! :-)